index.js

```diff
@@ -504,9 +505,9 @@ const ContainerService = {
         userContextId = args.userContextId;
       }
 
-      const tab = browserWin.gBrowser.addTab(args.url || null, { userContextId });
+      const tab = browserWin.gBrowser.addTab(args.url || DEFAULT_TAB, { userContextId });
       browserWin.gBrowser.selectedTab = tab;
-      browserWin.focusAndSelectUrlBar();
+      browserWin.focusUrlBar();
```

Comment:
I prefer browserWin.focusAndSelectUrlBar(), plus it's not related with this PR. Separate commit? separate PR?